src/analysis/retail/shaman/restoration/modules/dragonflight/Tier31.tsx

```diff
@@ -0,0 +1,151 @@
+import ITEMS from 'common/ITEMS';
+import talents from 'common/TALENTS/shaman';
+import { TIERS } from 'game/TIERS';
+import Analyzer, { Options, SELECTED_PLAYER } from 'parser/core/Analyzer';
+import Events, { HealEvent } from 'parser/core/Events';
+import { isFrom4pcT31 } from '../../normalizers/CastLinkNormalizer';
+import { SHAMAN_T31_ID } from 'common/ITEMS/dragonflight';
+import { SpellLink } from 'interface';
+import ItemSetLink from 'interface/ItemSetLink';
+import STATISTIC_CATEGORY from 'parser/ui/STATISTIC_CATEGORY';
+import STATISTIC_ORDER from 'parser/ui/STATISTIC_ORDER';
+import Statistic from 'parser/ui/Statistic';
+import { formatNumber, formatPercentage } from 'common/format';
+import ItemHealingDone from 'parser/ui/ItemHealingDone';
+import RiptideTracker from '../core/RiptideTracker';
+import Combatants from 'parser/shared/modules/Combatants';
+
+const RIPTIDE_4P_POWER = 0.2; // Riptide bonus power = 25% => Tier set value = 25%/125% = 20% of total amount
```

Comment:
25%

Reply:
It is not confusing but you are writing unnecessary code instead of using the helper method it i put in the example code that does all this for you